<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListAvailable</class>
 <widget class="QDialog" name="ListAvailable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QListWidget" name="lw_items">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="cb_list_filter">
         <item>
          <property name="text">
           <string>name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>version</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>company</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vendor</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_list_filter">
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_list_filter">
         <property name="styleSheet">
          <string notr="true">padding: 4px</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../resources/icons.qrc">
           <normaloff>:/icons/icons/search.svg</normaloff>:/icons/icons/search.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="cb_list_branches"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
